docker/ludwig-gpu/Dockerfile

```diff
@@ -9,6 +9,8 @@
 #   model serving
 #
 
+FROM python:3.8.12-slim
```

Comment:
We should see what version of Python is used by the parent pytorch image.

Reply:
pytorch docker is using `Python 3.7.11`, does this block us from upgrading?